### ISSUE TITLE
Does Windows 10 work?

### ISSUE THREAD

**Issue by seanthegeek** [spender-sandbox/cuckoo-modified#336]
Has anyone tried Windows 10 lately. IIRC cuckoomon doesn't work with it, but I'd be happy to be wrong :)

**Comment by marek2003** [comment]
take a look here https://github.com/spender-sandbox/cuckoo-modified/issues/74 i dunno if it helps.
regards

**Comment by hackdefendr** [comment]
Yes it works as long as you fully disable Windows Defender, Windows Firewall, and Windows UAC.
